app/src/main/java/src/Pieces/Rook.java

```diff
@@ -3,14 +3,21 @@
 import src.PiecesMoves.LinedMove;
 import src.Utilities.Position;
 
-public class Rook extends Piece {
-    private final char figure = 'R';
+public class Rook extends Piece{
+    private char figure = 'R';
+    private final String WHITE_ROOK_DIR = "src/main/java/resources/WhiteRook.png";
+    private final String BLACK_ROOK_DIR = "src/main/java/resources/BlackRook.png";
     /**
      *Class to start a bishop.
      */
     public Rook(final boolean white, final Position position) {
         super(white, position);
         addMoveType(new LinedMove());
+        if (white == true) {
+            setImage(WHITE_ROOK_DIR);
+        } else {
+            setImage(BLACK_ROOK_DIR);
+        }
```

Comment:
Other way to set this condition could be white? setImage(WHITE_ROOK_DIR):setImage(BLACK_ROOK_DIR), maybe its better this way?